### PR TITLE
DO NOT SUBMIT - Design Doc Test

### PR DESCRIPTION
**Pull request by ianegordon**
Do not merge.

This is a test of the capabilities of GitHub to determine if we can publish design docs for new APIs and features as markdown in a PR.

### CODE REVIEW

File: docs/donotsubmit.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+
+* Onion
+
+* Chicken
+
+* Andouille Sausage
+
+* Shrimp
+
+## Preperation
```
Comment (ianegordon) [inline docs/donotsubmit.md]: s/Preperation/Preparation/

File: docs/donotsubmit.md [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+
+* Heat dry flour in dutch oven, stirring constantly, until it gains some color
+
+* Add oil and combine into a roux
```
Comment (jverkoey) [inline docs/donotsubmit.md]: What kind of oil?

**Review by jverkoey — CHANGES_REQUESTED** [review]

File: docs/donotsubmit.md (hunk 1 above)
Comment (ianegordon) [reply docs/donotsubmit.md]: Done.

File: docs/donotsubmit.md (hunk 2 above)
Comment (ianegordon) [reply docs/donotsubmit.md]: Updated ingredient list above.

**Review by jverkoey — APPROVED** [review]

**Comment by ianegordon** [comment]
Final thoughts:

PROs:
Open
Changes are tracked as commit w/ history
Design Doc could live in the same PR as the API review
Docs would be in markdown from the start

CONs:
Switching to the edit interface hides comments
The 'threading' of comments isn't quite there yet, making discussions difficult.

**Pull request closed without merging**